tests/integration/collection_description/updates/add/collections_test.go

```diff
@@ -74,7 +74,7 @@ func TestColDescrUpdateAddCollections_Errors(t *testing.T) {
 						{ "op": "add", "path": "/2", "value": {"ID": 2, "Name": "Dogs"} }
 					]
 				`,
-				ExpectedError: "adding collections via patch is not supported. ID: 2",
+				ExpectedError: "schema name can't be empty",
```

Comment:
todo: Sorry, whilst my last comment was misplaced, this error *was* useful to the user, please revert this test and change the production code to accomodate.

Reply:
Done